lib/plugins/features/movies/index.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+exports.register = (server, options, next) => {
+
+  server.route([{
+    method: 'POST',
+    path: '/movies',
+    config: {
+      handler: (request, reply) => {
+        reply('hello world');
```

Comment:
I know it says in Notion for you to create this PR after setting up the endpoint, but leaving the endpoint to only return `hello world` is most likely undesired behavior. Usually when we setup endpoints for production-level code, we opt to combine the steps of setting up the endpoint, its related controller, the validators for request payloads or query params, and the unit + integration tests (as long as it's less than ~400 LOC for anything that isn't for testing because large PRs are harder for reviewers to parse in one go).

What do you think about implementing the controller portion here? If you still want granularity and want to split up the PR, perhaps another way would be to add the validator here and then have a separate PR for the controller + integration tests. Let me know what you think!

Reply:
That makes sense. I think I will include the validator and controller (with unit tests) portions here. I'll leave the integration test for a separate PR.

Reply:
ok should be good to go. things got a little weird because i was working off the validators branch at first. i then rebased off that branch and continued working on the controllers in this branch. let me know if anything looks strange wrt that. i also installed a linter for vscode so hopefully i'll be able to catch things before they go through the ci pipeline.

Reply:
I believe running `yarn lint` and/or `yarn lint --fix` locally should work if you want to catch syntax issues and whatnot before committing and pushing to this branch. An editor's linter will be good for catching linting issues on the fly, but there might be a slight discrepancy where if it uses a different linter than what we employ (`eslint`).

Another tip would be to potentially move the `yarn lint` step in the `.travis.yml` before the `yarn test` step so that we can fast fail. This is especially relevant in our larger codebases where we have thousands upon thousands of tests which take upwards of 10-15 minutes to run, wherein a linting issue caught at the end of successful tests would require us to fix that issue, re-deploy, then wait another 10-15 minutes